module/Calculator/Pid.cpp

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file Pid.cpp
+ * @brief PIDを計算するクラス
+ * @author TakumaChihaya
```

Comment:
```suggestion
 * @author CHIHAYATAKU
```

前年は本名ではなくGitHubでの名前に統一してたと思うので、細かいですがこのようにお願いします🙇‍♂️

Reply:
すみません。直した気になってました🙇